refurbish/tests/Main.hs

```diff
@@ -157,25 +161,47 @@ strategies = [ R.LayoutStrategy allocator grouping trampoline
                              ]
              ]
 
-mkRewriteTests :: C.HandleAllocator -> TS.Sweets -> Natural -> TS.Expectation
+mkRewriteTests :: String -> C.HandleAllocator
```

Comment:
Should we parse the level into a number here?

Reply:
I'm doing direct comparisons, nothing `Ord`-ish.  We could, but I didn't need it yet.